lib/src/state.dart

```diff
@@ -484,17 +494,31 @@ class EntityManager implements EntityObserver {
 
   /// Returns the component instance or `null`.
```

Comment:
please fix the comment, it should be `Returns the component instance or throws an exception if component is null`